step-templates/k8s-deploy-kustomize.json

```diff
@@ -0,0 +1,58 @@
+{
+  "Id": "4b0be8ea-e191-4e27-9294-6cc56daaa488",
+  "Name": "Kubernetes - kustomize template deployment",
+  "Description": "Load package with kubernetes configuration and deploy to cluster with kustomize evaluation.",
```

Comment:
Some thoughts on the overall name and description:
- Since this is a `bash script`, it is worth considering either adding that it's a bash script in the description, or in the title e.g. `Kubernetes - kustomize template deployment (bash)`
- Also for users who don't know what kustomize is, a link to the documentation would also be helpful.
- The script checks for kubectl - the description should state this is a requirement to help users understand pre-requisites.